src/main/java/liquibase/ext/mongodb/statement/CountCollectionByNameStatement.java

```diff
@@ -50,4 +50,9 @@ public long queryForLong(final MongoConnection connection) {
                 .filter(s -> s.equals(getCollectionName()))
                 .count();
     }
+
```

Comment:
A correct implementation of `toJs` for the count method.
Is needed since I made the baseclass method abstract

Reply:
This is actually counting collections not documents

Reply:
will fix in a subsequent commit

Reply:
Sorry for the misunderstanding. I see now how this is used and is different from `CountDocumentsInCollectionStatement`.
This could only ever return zero or one, so could potentially be made clearer by refactoring it to implement a new interface such as `NoSqlQueryForBooleanStatement`:
```
public boolean exists(MongoConnection connection);
```

Thoughts?

Reply:
Yes, could worth, it's just I tend to stay similar with Executor methods from liquibase core, otherwise can get into unsupported features when new Liquibase release.